TurkishFilter/sections/specific.txt

```diff
@@ -2308,10 +2309,10 @@ haberler.com###DetailMtrxCentre > div#quiz_masthead[style^="width:970px; height:
 ||haber50.com/*/images/banner/
 freemagazine.com.tr##.u-ads
 freemagazine.com.tr##.sagads
-onlinedizi.net##.video_top_banner
-onlinedizi.net##body > a[href][target="_blank"][rel="nofollow"]
-||onlinedizi.net/images/video_top.js
-||onlinedizi.net/images/betwinner-bg.jpg
+onlinedizi.tv##.video_top_banner
```

Comment:
No elements and requests - obsolete rules:

onlinedizi.tv##.video_top_banner
onlinedizi.tv##body > a[href][target="_blank"][rel="nofollow"]
||onlinedizi.tv/images/video_top.js
||onlinedizi.tv/images/betwinner-bg.jpg